eng/pipelines/templates/stages/archetype-java-release-pom-only.yml

```diff
@@ -66,14 +66,9 @@ stages:
                 deploy:
                   steps:
                     - checkout: self
-                    - template: /eng/pipelines/templates/steps/stage-artifacts.yml
-                      parameters:
-                        SourceFolder: ${{parameters.ArtifactName}}-signed
-                        TargetFolder: ${{artifact.safeName}}
-                        PackageName: ${{artifact.name}}
                     - template: /eng/common/pipelines/templates/steps/create-tags-and-git-release.yml
                       parameters:
-                        ArtifactLocation: $(Pipeline.Workspace)/${{artifact.safeName}}
+                        ArtifactLocation: ${{parameters.ArtifactName}}-signed/${{artifact.groupId}}/${{artifact.name}}
```

Comment:
@mitchdenny shouldn't the artifact location be `$(Pipeline.Workspace)/${{parameters.ArtifactName}}-signed/${{artifact.groupId}}/${{artifact.name}}` like it is in eng/pipelines/templates/stages/archetype-java-release.yml?

Reply:
Fixed